packages/react-notifications/src/components/notification-kinds/api-error-message/api-error-message.spec.tsx

```diff
@@ -96,10 +96,12 @@ describe('render', () => {
   });
   it('should show message for DuplicateField', () => {
```

Comment:
What about testing for both cases?

Reply:
> What about testing for both cases?

I've added cases for REST and GraphQL errors 👍